docs/users_guide/exts.rst

```diff
@@ -21,7 +21,6 @@ Language extensions
     exts/constraints
     exts/type_signatures
     exts/bindings
-    exts/linear_types
```

Comment:
I hadn't seen it in the other place. My bad but adding it here made the link to linear types, in the index clickable. I've also renamed `linear-types` to `linear_types`, so maybe there was a typo? Do double-check that the index works.

Reply:
Do you mean present and clickable in `index.html`? Yes, it's still there.